packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_services.dart

```diff
@@ -118,6 +118,19 @@ class DeepLinksServices {
     );
   }
 
+  Future<Map<String, List<DomainError>>> validateIosDomain({
+    required List<String> domains,
+    required String applicationId,
+    required String? localFingerprint,
+  }) async {
+    //TODO: This is just a fake response, implement the iOS domain validation when the API is available.
```

Comment:
TODO format should be `// TODO(username or github issue): description`. Otherwise, we may trigger lints in google3 that check for this.